(DOCSP-28658) Document new Compensating Write error - Node.js (#3011)
mongodb/docs-realm: source/sdk/node/sync/handle-sync-errors.txt

   :depth: 3
   :class: singlecol

When you use Atlas Device Sync in your Realm app, you can encounter a new class
of errors: :ref:`sync errors <sync-errors>`.

The Realm Node.js SDK can help you detect and handle sync errors. For example,
you can write your own sync error handler to respond to specific errors. You can
also define how your client app handles client resets.

.. _node-set-error-handler:

Sync Error Handler
------------------

> You should set an error handler for apps that use Atlas Device Sync. A generic

Reviewer: Style nit: Avoid subjunctive
```suggestion
As best practice, set an error handler for apps that use Atlas Device Sync. A generic
```

Author: I'm going to leave "should", as the style guide suggests that its synonymous with "best practice". https://www.mongodb.com/docs/meta/style-guide/writing/use-effective-verbs/#use-modal-verbs-accurately

.. include:: /includes/sync-errors-in-app-services.rst

Handle Compensating Write Errors
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

You may want your sync error handler to specifically address compensating write
errors in a way that makes sense for your app. The :js-sdk:`CompensatingWriteError 
<classes/CompensatingWriteError.html>` class can help you identify and react to

> compensating write erros in your custom error handler.

Reviewer: ```suggestion
compensating write errors in your custom error handler.
```
typo

Author: Thank you for catching this!
